cdts/pam-alma9-x86_64/meta.yaml

```diff
@@ -29,20 +29,20 @@ requirements:
     - libeconf-conda-x86_64 >=0.3.5 *_{{ cdt_build_number|int + 1000 }}
     - libpwquality-conda-x86_64 >=0.9.9 *_{{ cdt_build_number|int + 1000 }}
     - libselinux-conda-x86_64 >=1.33.2 *_{{ cdt_build_number|int + 1000 }}
-    - openssl-conda-x86_64 >=3.0.0 *_{{ cdt_build_number|int + 1000 }}
+    - openssl 3.*
   host:
     - audit-libs-conda-x86_64 >=1.0.8 *_{{ cdt_build_number|int + 1000 }}
     - libeconf-conda-x86_64 >=0.3.5 *_{{ cdt_build_number|int + 1000 }}
     - libpwquality-conda-x86_64 >=0.9.9 *_{{ cdt_build_number|int + 1000 }}
     - libselinux-conda-x86_64 >=1.33.2 *_{{ cdt_build_number|int + 1000 }}
-    - openssl-conda-x86_64 >=3.0.0 *_{{ cdt_build_number|int + 1000 }}
+    - openssl 3.*
     - sysroot_linux-64 2.34.*
```

Comment:
I'm not sure if this is functional yet; since the CDTs are noarch, they can be depended on easily also in a cross-compilation scenario, but then just depending on `openssl` wouldn't be correct I think.

My thinking was that we'd want to avoid CDTs for openssl (which I still think is reasonable), but perhaps it'd be better to simply remove `openssl` as a dependency here? Another alternative would be to build noarch `openssl_linux-64` etc.